media/css/firefox/whatsnew/includes/_loading-placeholder.scss

```diff
@@ -0,0 +1,57 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+@keyframes loading {
+    0% {
+        background-position: 100% 50%;
+    }
+
+    100% {
+        background-position: 0 50%;
+    }
+}
+
+.c-loading-placeholder {
+    min-height: 900px;
+}
+
+.c-loading-placeholder-element {
+    animation: loading 1.2s ease-in-out infinite;
```

Comment:
**praise:** love that you incorporated this placeholder into the pages. You did a good job coding it out too!  🚀